src/NuGet.Clients/PackageManagement.PowerShellCmdlets/PackageActionBaseCommand.cs

```diff
@@ -103,7 +101,7 @@ protected override void ProcessRecordCore()
             {
                 if (ex.InnerException is PackageAlreadyInstalledException)
                 {
-                    Log(ProjectManagement.MessageLevel.Info, ex.Message);
+                    Log(MessageLevel.Info, ex.Message);
```

Comment:
Not in this case, it is info level since the code knows exactly what exception is being thrown and has a particular idea of what message should be logged.